lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -92,6 +92,149 @@ html.video-fullscreen {
       display: inline;
     }
   }
+
```

Comment:
+1 Yeah I saw that this was going into your feature branch. 

(And yeah you could make sure to nest lines 99 through 235 inside of the following:  .timer-bar-class-scope, .proctoring-message-class-scope { }) 

Obviously those are the wrong class names, but that's at least 1 way to make sure this only impacts those parts of the UI. 

Another approach would actually be to use the _developer.scss partial file for styles we know may not be 100% Sass styling convention friendly, or would require potentially uncharted, unknown, or substantial sass cleanup. This also makes is really easy for UX to track cases where we know we have styles that need some further cleanup. :)

Thanks!